src/lib/fragments/AgGrid.react.js

```diff
@@ -192,15 +194,66 @@ export default class DashAgGrid extends Component {
 
     setSelection(selection) {
         const {gridApi} = this.state;
+        const {getRowId} = this.props;
         if (gridApi && selection) {
-            if (!selection.length) {
+            this.setState({pauseSelections: true});
+            if (Object.keys(selection).includes('function')) {
```

Comment:
```suggestion
            if (has('function', selection)) {
```
Just a little cleaner & faster :)